resources/css/theme.css

```diff
@@ -140,6 +140,7 @@ sp-theme {
     --spectrum-accent-color-1100: #225E22;
     --spectrum-accent-color-1200: #1A541A;
     --spectrum-accent-color-1300: #134913;
+    --spectrum-component-height-100: 0px;
```

Comment:
@ancy2023 , we can't set the spectrum button height to 0 as that causes the "hidden apps" button to no longer display. Please try other options.

Reply:
@ancy2023, are you still working on this issue?